data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -99,28 +101,63 @@ def transform(self) -> None:
             inplace=True,
         )
 
+        # Only use disasters linked to climate change
+        disaster_categories = [
+            "AVLN",  # Avalanche
+            "CFLD",  # Coastal Flooding
+            "CWAV",  # Cold Wave
+            "DRGT",  # Drought
+            "HAIL",  # Hail
+            "HWAV",  # Heat Wave
+            "HRCN",  # Hurricane
+            "ISTM",  # Ice Storm
+            "LNDS",  # Landslide
+            "RFLD",  # Riverine Flooding
+            "SWND",  # Strong Wind
+            "TRND",  # Tornado
+            "WFIR",  # Wildfire
+            "WNTW",  # Winter Weather
+        ]
+
+        # Some disaster categories do not have agriculture value column
+        agri_columns = [f"{x}_EALA" for x 
+        in disaster_categories if f"{x}_EALA"in list(df_nri.columns)]
+
+        population_columns = [f"{x}_EALP" for x 
+            in disaster_categories if f"{x}_EALP" in list(df_nri.columns)]
+
+
+        buildings_columns = [f"{x}_EALB" for x 
+            in disaster_categories if f"{x}_EALB" in list(df_nri.columns)]
+
+        disaster_population_sum_series = df_nri[
+            population_columns
+        ].sum(axis=1)
+
+        disaster_agriculture_sum_series = df_nri[
+            agri_columns
```

Comment:
This is super nitpicky, so feel free to ignore, but for readability do we want to make sure all the variable names match with the same way of describing the category? 

E.g., `disaster_agriculture_sum_series` and `agriculture_columns` might be more consistent than `agri_columns`. 

And `disaster_building_sum_series` has singular building compared to `buildings_columns`. 

But feel free to ignore, this is super nitpicky.